modules/ROOT/pages/permissions-by-product.adoc

```diff
@@ -52,8 +52,11 @@ Or:
 
 == Design Center
 
+* Design Center Viewer
+* Design Center Creator
 * Design Center Developer
 
+
```

Comment:
We may need to have a similar division as Exchange below, with:

Organization level permissions: 
* Design Center Viewer
* Design Center Creator
* Design Center Developer

Project level permissions:
* Project administrator
* Project editor
* Project viewer

Reply:
Agree with Leandro here! We should call out both org level and project level permissions! Thank you